### DISCUSSION TITLE
pesudo labeling

### DISCUSSION THREAD

**Discussion by kmouleejunhyuk** [boostcampaitech2/object-detection-level2-cv-06#114] [ideas]
저번 대회에서 성능 향상에 사용한 기법인 pesudo labeling을 실험해 볼 생각입니다
지금 inference시 bbox output이 엄청 많은데 일부만 (confidence 기준) 학습해 볼 생각입니다.

**Comment by jaehwan-AI** [comment]
pesudo labeling을 하려면 싱글 모델의 성능이 어느정도 나와야 하는 것 아닌가요?

**Reply by kmouleejunhyuk** [reply]
@jaehwan-AI 대회 종료도 얼마 안남아서 슬슬 시도해볼 떄라고 생각했습니다. 더 나중에 해봐야 할까요?

**Reply by kwakjeeyoon** [reply]
현재 나오는 성능도 낮은 성능은 아니고 pesudo labeling 관련 작업을 미리 해놓는것도 의미가 있다고 생각합니다!

**Reply by jaehwan-AI** [reply]
음.. 제가 잘못 생각한 것 같네요. 시간이 얼마 안남았으니 다 해봅시다
